src/main/java/billgates/Main.java

```diff
@@ -13,14 +13,19 @@
 import billgates.usecases.delete_entry.DeleteEntryUseCase;
 import billgates.view.gui.MainFrame;
 
+import javax.swing.*;
+import javax.swing.plaf.metal.MetalLookAndFeel;
+
 /**
  * Clean Architecture Layer: Frameworks & Drivers
  *
  * @author Scott
  */
 public class Main {
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws UnsupportedLookAndFeelException {
+        UIManager.setLookAndFeel(new MetalLookAndFeel());
```

Comment:
We could try and catch the exception. 

If the exception happened, we can use the default look and feel.